lib/sort_by_length.rb

```diff
@@ -1,7 +1,25 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2), because there are two nested loops that are
+# both dependent on the size of the input.
+# Space complexity: O(n), because there is a single array made during
+# the process.
 def sort_by_length(my_sentence)
```

Comment:
Nice bubblesort, but since you're doing it in place the space complexity is O(1), not O(n)